fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/MinimumDaysBetweenDisbursalAndFirstRepaymentTest.java

```diff
@@ -43,7 +43,7 @@
 /**
  * Test the creation, approval and rejection of a loan reschedule request
  **/
-@SuppressWarnings({ "rawtypes" })
+@SuppressWarnings({ "rawtypes", "AbbreviationAsWordInName" })
 public class MinimumDaysBetweenDisbursalAndFirstRepaymentTest {
```

Comment:
What was the reason for suppressing for this file? I didn't see anything specific why we would not be able to use... but perhaps I missed something

Reply:
Agreed, Removed.